src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -265,6 +265,40 @@ protected function addMutatedAttributesToArray(array $attributes, array $mutated
         return $attributes;
     }
 
+    /**
+     * Check if model has a given attribute.
+     *
+     * @param  string $key
+     * @return bool
+     */
+    public function hasAttribute($key)
+    {
+        // If the attribute exists in the attribute array or has a "get" mutator we will
+        // return the attribute's value. Otherwise, we will proceed as if the developers
+        // are asking for a relationship's value. This covers both types of values.
```

Comment:
I think that comment doesn't make any sense here and should be moved back to where it was.

Reply:
Fixed